live_site.py

```diff
@@ -0,0 +1,433 @@
+		ip_address = str(ip)  
+
+		# This adds/creates the domain name
+		new_domain = add_new_domain(domain_name, ip_address)
+
+		if new_domain is not None:
+			click.secho('>>> Your new domain name {} has been created with IP Address {}. '.format(domain_name, ip_address), fg='bright_green')
+		else:
+			click.secho('[!] Request Failed', fg='bright_red')
+
+
+def create_domain(droplet, domain_name):
+
+	domain_exists = check_domain(domain_name)
+	click.echo(domain_exists)
+
+	if domain_exists == domain_name:
+		click.secho('>>> Your domain name {} already exists!'.format(domain_name), fg='bright_yellow')
+		click.secho('>>> We are now removing it to droplet IP mapping ...', fg='bright_yellow')
+		del_domain(domain_name)
+		click.secho('Creating a new domain name now...', fg='bright_yellow')
+		will_add_domain(droplet, domain_name)
+	else:
+		click.echo('...')
+		will_add_domain(droplet, domain_name)
+
+
+def add_new_domain_record(type, name, data, ttl, domain_name):
+
+	api_url = '{0}domains/{1}/records'.format(api_url_base,domain_name)    
+
+	records = {"type":type,"name":name,"data":data,"ttl":ttl}
+
+	response = requests.post(api_url, headers=headers, json=records) 
+
+	if response.status_code == 201:
+		records = json.loads(response.content)
+		return records
+	else:
+		click.secho('[?] Unexpected Error: [HTTP {0}]: Content: {1}'.format(response.status_code, response.content), fg='bright_red')
+		return None                     
+		
+
+def add_domain_rec(droplet, domain_name):
+	# Get the IP Address associated with jediluke.space
+	ip = get_hostname_ip(droplet)
+	rec_data = str(ip)
+
+	add_response = add_new_domain_record('A','www',rec_data,1800, domain_name)
+
+	if add_response is not None:
+		click.secho('>>> Your A Record has been created... ', fg='bright_green')
+		for k, v in add_response.items():
+			click.secho('  {0}:{1}'.format(k, v), fg='bright_green')
+	else:
+		click.secho('[!] Request Failed', fg='bright_red')
+
+def set_nginx(droplet, domain_name):
+
+	host = get_hostname_ip(droplet)
+
+	client = SSHClient(host, user='root', password='', pkey=ssh_key,port=22)
+	click.secho('Connecting to your server...', fg='bright_yellow')
+
+	def upload():
+	# This funtion will:
+	# Upload the initial files and configuration needed in setting up Nginx to the server   
+
+		click.echo('Setting up your configuration block for {}...'.format(domain_name))
+		loc_file1 = 'D:/Workbench/Luke/jediluke.space/files/jediluke.space.txt'
+		rem_file1 = '/etc/nginx/jediluke.space.txt'
+		loc_file2 = 'D:/Workbench/Luke/jediluke.space/files/nginx.conf'
+		rem_file2 = '/etc/nginx/nginx.conf'
+
+		upload_1 = client.copy_file(loc_file1, rem_file1)
+		upload_2 = client.copy_file(loc_file2, rem_file2)
+
+		cmd = ["sudo mv /etc/nginx/jediluke.space.txt /etc/nginx/sites-available/jediluke.space", "sudo ln -s /etc/nginx/sites-available/jediluke.space /etc/nginx/sites-enabled/", "sudo nginx -t", "sudo systemctl restart nginx"]
```

Comment:
when we have lots of commands, try putting them all on their own line - it's a list, so Python wont mind, and it just looks better rearranging it a little.

Your code is your own creation, and once you get more used to it, code can even look beautiful!! So feel free to experiment and try to make the most beautiful code you can!